### PR TITLE
Create Fact “stiff-leaved-goldenrod”

### PR DESCRIPTION
**Pull request by NicolasRobertson**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/facts/stiff-leaved-goldenrod-1.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  license: CC by 2.0
+  url: https://www.flickr.com/photos/acryptozoo
+source:
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+  name: "Bees: An Identification and Native Plant Forage Guide"
+category: Habitat
+---
+<!--StartFragment-->
+
+**This species is more commonly visited by female bees than male, due to it flowering earlier than other goldenrod species (bee book,p. 185)**
```
Comment (floogulinc) [inline src/facts/stiff-leaved-goldenrod-1.md]: Be careful just pasting in from the document

File: src/facts/stiff-leaved-goldenrod-1.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+title: Stiff-leaved Goldenrod
+image:
+  src: ../images/stiff-leavedgoldenrod.jpg
+  alt: Stiff-leaved Goldenrod
+  creator: Andrew Cannizzaro
+  license: CC by 2.0
+  url: https://www.flickr.com/photos/acryptozoo
+source:
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+  name: "Bees: An Identification and Native Plant Forage Guide"
```
Comment (floogulinc) [inline src/facts/stiff-leaved-goldenrod-1.md:11]: No quotes on this please

**Review by floogulinc — CHANGES_REQUESTED** [review]

File: src/facts/stiff-leaved-goldenrod-1.md (hunk 1 above)
Comment (floogulinc) [reply src/facts/stiff-leaved-goldenrod-1.md]: It is better to right click and "paste as plain text" and don't include the "(bee book,p. 185)". This shouldn't be bold.

**Comment by floogulinc** [comment]
I think this can be combined with #68

File: src/facts/stiff-leaved-goldenrod-1.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Stiff-leaved Goldenrod
+image:
+  src: ../images/stiff-leavedgoldenrod.jpg
+  alt: Stiff-leaved Goldenrod
+  creator: Andrew Cannizzaro
+  license: CC by 2.0
+  url: https://www.flickr.com/photos/acryptozoo
```
Comment (floogulinc) [inline src/facts/stiff-leaved-goldenrod-1.md]: Please link to the specific image

**Review by floogulinc — CHANGES_REQUESTED** [review]

File: src/facts/stiff-leaved-goldenrod-1.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+image:
+  src: ../images/stiff-leavedgoldenrod.jpg
+  alt: Stiff-leaved Goldenrod
+  creator: Andrew Cannizzaro
+  license: CC by 2.0
+  url: https://www.flickr.com/photos/acryptozoo/21143130290
+source:
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+  name: "Bees: An Identification and Native Plant Forage Guide"
+category: Plant
```
Comment (floogulinc) [inline src/facts/stiff-leaved-goldenrod-1.md]: Haven't we put the other plants in the "habitat" category?

File: src/facts/stiff-leaved-goldenrod-1.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+---
+Preferring well-drained soil, stiff-leaved goldenrod is common along roadsides and dry prairies.
+
+This species is more commonly visited by female bees than male, due to it flowering earlier than other goldenrod species.
```
Comment (floogulinc) [inline src/facts/stiff-leaved-goldenrod-1.md]:
```suggestion
Preferring well-drained soil, stiff-leaved goldenrod is common along roadsides and dry prairies. This species is more commonly visited by female bees than male due to flowering earlier than other goldenrod species.
```

**Review by floogulinc — CHANGES_REQUESTED** [review]

File: src/facts/stiff-leaved-goldenrod-1.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+title: Stiff-leaved Goldenrod
+image:
+  src: ../images/stiff-leavedgoldenrod.jpg
+  alt: Stiff-leaved Goldenrod
+  creator: Andrew Cannizzaro
+  license: CC by 2.0
```
Comment (floogulinc) [inline src/facts/stiff-leaved-goldenrod-1.md]:
```suggestion
  license: CC BY 2.0
```

**Review by floogulinc — CHANGES_REQUESTED** [review]

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please rename the file to remove the `-1`

**Review by kklamberty — CHANGES_REQUESTED** [review]
Please see the previous comment from @floogulinc and act on it.

**Review by floogulinc — APPROVED** [review]
The category is due to change when we switch to tags anyway so it's not important habitat versus plant.

**Review by kklamberty — APPROVED** [review]

**Pull request merged**